cmake/FindHYPRE.cmake

```diff
@@ -0,0 +1,20 @@
+find_package(PkgConfig QUIET)
```

Comment:
I am not sure if we even need this file at all.

Reply:
Agreed - removed